src/crimson/osd/pg_backend.cc

```diff
@@ -119,18 +119,22 @@ PGBackend::mutate_object(
   std::set<pg_shard_t> pg_shards,
   crimson::osd::ObjectContextRef &&obc,
   ceph::os::Transaction&& txn,
-  const MOSDOp& m,
+  OSDOpParams&& osd_op_p,
   epoch_t min_epoch,
   epoch_t map_epoch,
-  eversion_t ver)
+  vector<pg_log_entry_t> logv)
 {
+  auto& m = osd_op_p.req;
```

Comment:
please move this change to the commit which uses this variable. and please define a local variable to the place where it's used . see https://google.github.io/styleguide/cppguide.html#Local_Variables